---
string[] joints_name
float64[] posj
float64[] posx

